oxd-common/src/main/java/org/gluu/oxd/common/ErrorResponseCode.java

```diff
@@ -101,7 +101,8 @@
     INVALID_AUTHORIZATION_OXD_ID(400, "invalid_authorization_oxd_id", "`oxd_id` in `AuthorizationOxdId` header is invalid. The `AuthorizationOxdId` header should contain `oxd_id` from `protect_commands_with_oxd_id` field in oxd-server.yml."),
     AUTHORIZATION_OXD_ID_NOT_FOUND(400, "authorization_oxd_id_not_found", "`oxd_id` in `AuthorizationOxdId` header is not registered in oxd."),
     NO_CLIENT_ID_RETURNED(500, "no_client_id_returned", "`client_id` is not returned from OP host. Please check OP log file for error (oxauth.log)."),
-    NO_CLIENT_SECRET_RETURNED(500, "no_client_secret_returned", "`client_secret` is not returned from OP host. Please check: 1) OP log file for error (oxauth.log) 2) whether `returnClientSecretOnRead` configuration property is set to true on OP host.");
+    NO_CLIENT_SECRET_RETURNED(500, "no_client_secret_returned", "`client_secret` is not returned from OP host. Please check: 1) OP log file for error (oxauth.log) 2) whether `returnClientSecretOnRead` configuration property is set to true on OP host."),
+    OXD_ACCESS_DENIED(500, "oxd_access_denied", "The caller is not allowed to make request to oxd. To allow add ip_address of caller in `bind_ip_addresses` in field of `oxd-server.yml`.");
```

Comment:
Status code should be 403, it's not internal error.

Reply:
Done